WooCommerce/Classes/Extensions/UITabBar+Appearance.swift

```diff
@@ -10,8 +10,13 @@ extension UITabBar {
         let appearance = Self.appearance()
         appearance.barTintColor = .appTabBar
         appearance.tintColor = .text
+        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.splitViewInOrdersTab) {
+            // tab bar needs to be translucent to get rid of the extra space at the bottom of
+            // the view controllers embedded in split view.
+            appearance.isTranslucent = true
```

Comment:
If I understand correctly, the tab bar is still not visually translucent (at least from my testing) and setting `isTranslucent = true` here is just to get rid of the workaround for the offline banner?

Reply:
This is true, the bar still looks opaque since I'm keeping the [appearance setup](https://github.com/woocommerce/woocommerce-ios/blob/abddae029a69d8fcc37b4ab1f68fa0ba6acd3d62/WooCommerce/Classes/Extensions/UITabBar%2BAppearance.swift#L36) as-is. I'm not sure yet how we should update it, maybe this should be handled in a separate issue.

Reply:
Sounds good! I asked about the translucent material design update https://github.com/woocommerce/woocommerce-ios/issues/1661 in p1647576625846659-slack-C0354HSNUJH